jdi-light/src/main/java/com/epam/jdi/light/logger/JDILogger.java

```diff
@@ -116,18 +133,33 @@ public String getName() {
         return name;
     }
 
-    private static void writeToAllure(String message) {
+    private static void writeToAllure(String message, Status status) {
         if (!writeToAllure) return;
         final String uuid = UUID.randomUUID().toString();
-        StepResult step = new StepResult().withName(message).withStatus(PASSED);
+        StepResult step = new StepResult().withName(message).withStatus(status);
         getLifecycle().startStep(uuid, step);
+        if (FAILED.equals(status)) {
+            if ("on fail".equals(screenshotStrategy)) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [These nested if statements could be combined](https://app.codacy.com/app/42_/jdi-light/pullRequest?prid=4558480)